TrueNASCORE/TrueNASCORE.php

```diff
@@ -109,14 +109,17 @@ public function uptime($inputSeconds)
 
 	public function alerts($alert)
 	{
-		$count = 0;
+		$count_total = $count_critical = 0;
 		foreach ($alert as $key => $value) {
 			if ($value["dismissed"] == false) {
-				$count += 1;
+				$count_total += 1;
+				if (!in_array($value["level"], array("NOTICE", "INFO"))) {
```

Comment:
Not being hindered by knowledge of the TrueNAS api output...

You use `!in_array($value["level"], array("NOTICE", "INFO"))` which is obviously not the same as `in_array($value["level"], array("CRITICAL"))` because it counts everything except NOTICE and INFO... however, the UI you created suggests to the user that **only** CRITICAL alerts are counted.

I can imagine that might lead to confusion for some.

Reply:
You are right, my use of the term CRITICAL is misleading. There is a range of alert types in the API output, and for me personally, all types except NOTICE or INFO are the ones I want to check out immediately. For example, a scrub or replication finished is not so interesting as a drive failure.